Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/validation/ValidationResult.java

```diff
@@ -0,0 +1,24 @@
+package pg.autyzm.friendly_plans.validation;
+
+public class ValidationResult {
+
+    private ValidationStatus validationStatus;
```

Comment:
I'd add final to this two fields to ensure this class stay immutable in the future